--- a/quickshell/calendar/CalendarPopup.qml
+++ b/quickshell/calendar/CalendarPopup.qml
@@ -428,7 +428,7 @@
         Rectangle {
             anchors.fill: parent
             radius: ThemeBackend.borderRadius
-            color: window.base
+            color: Qt.rgba(window.base.r, window.base.g, window.base.b, OpacityExt.calendar)
             border.color: window.surface0
             border.width: 1
             clip: true
