--- a/quickshell/quickactions/Floating.qml
+++ b/quickshell/quickactions/Floating.qml
@@ -1077,7 +1077,7 @@
                         height: floatingWidget.outerCornerRadius
                         preferredRendererType: Shape.CurveRenderer
                         ShapePath {
-                            fillColor: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, 0.95)
+                            fillColor: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, OpacityExt.floating)
                             strokeColor: "transparent"
                             startX: 0
                             startY: 0
@@ -1101,7 +1101,7 @@
                         height: floatingWidget.outerCornerRadius
                         preferredRendererType: Shape.CurveRenderer
                         ShapePath {
-                            fillColor: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, 0.95)
+                            fillColor: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, OpacityExt.floating)
                             strokeColor: "transparent"
                             startX: 0
                             startY: floatingWidget.outerCornerRadius
@@ -1128,7 +1128,7 @@
                         bottomLeftRadius: 0
                         topRightRadius: floatingWidget.containerRadius
                         bottomRightRadius: floatingWidget.containerRadius
-                        color: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, 0.95)
+                        color: Qt.rgba(ThemeBackend.base.r, ThemeBackend.base.g, ThemeBackend.base.b, OpacityExt.floating)
                         border.width: 0
 
                         MouseArea {
